lib/clearance/password_strategies/sha1.rb

```diff
@@ -3,13 +3,21 @@ module PasswordStrategies
     module SHA1
       require 'digest/sha1'
 
+      DEPRECATION_MESSAGE = "[DEPRECATION] The SHA1 password strategy " \
```

Comment:
Yeah, I noticed this as well. I thought about it, but the "shared place" for that method to live would be the `PasswordStrategies` module which means adding code to present deprecations to parts of the app that are not being deprecated. I think I'm okay with the duplication here since it's never going to change once committed until the files are removed in 2.0.